source/chapters/library.rst

```diff
@@ -324,6 +326,14 @@ Tracks can be loaded into a deck in several ways:
   example, on macOS, dragging-and-dropping a track from iTunes onto one of
   Mixxx's waveform displays will load it into that deck.
 
+
+unLoading Tracks
+==============
+
```

Comment:
If we're adding a section for this, the eject buttons in the decks should also be mentioned.

Reply:
Thank you for the guide!